File: 0215-kth-largest-element-in-an-array/0215-kth-largest-element-in-an-array.cpp
class Solution {
public:
int findKthLargest(vector<int> &nums, int k)
{
    priority_queue<int, vector<int>, greater<int>> maxHeap;
    for (int i = 0; i < nums.size(); i++)
    {
        maxHeap.push(nums[i]);
        if (maxHeap.size() > k)
            maxHeap.pop();
    }
    return maxHeap.top();
}};

File: 0215-kth-largest-element-in-an-array/0215-kth-largest-element-in-an-array.java
class Solution {
  public int findKthLargest(int[] nums, int k) {
        PriorityQueue<Integer> queue=new PriorityQueue<>();
        for(int i=0;i<nums.length;i++){
            queue.add(nums[i]);
            while (queue.size()>k) {
                queue.poll();
            }
        }
        return queue.peek();
    }
}